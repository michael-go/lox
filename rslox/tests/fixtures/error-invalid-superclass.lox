var NotClass = "So not a class";
class OhNo < NotClass {}
