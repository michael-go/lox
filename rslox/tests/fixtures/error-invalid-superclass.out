# exit code: 65
# stdout:

# stderr:
Runtime error: Superclass must be a class.
[line 2] in <script>

